tests/functional/olp-cpp-sdk-authentication/ArcGisAuthenticationTest.cpp

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (C) 2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#include <olp/core/http/HttpStatusCode.h>
+#include <olp/core/porting/make_unique.h>
+#include "AuthenticationCommonTestFixture.h"
+#include "ArcGisTestUtils.h"
+#include "TestConstants.h"
+
+namespace {
+constexpr auto kErrorArcGisFailedCode = 400300;
+
+const std::string kErrorArcGisFailedMessage = "Invalid token.";
+
+}  // namespace
```

Comment:
you could place everything inside anonymous namespace and make it a bit easier for linker

Reply:
Ok, will do in a separate changes after I done with moving of all original tests